<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelDialog</class>
 <widget class="QDialog" name="ModelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>객체 탐지 모델 설정</string>
  </property>
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #f0f0f0;
    color: #333333;
}
QLabel {
    color: #333333;
}
QGroupBox {
    background-color: #f5f5f5;
    border-radius: 4px;
    border: 1px solid #c0c0c0;
    margin-top: 10px;
    color: #333333;
    font-weight: bold;
}
QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 5px;
}
QPushButton {
    background-color: #e0e0e0;
    color: #333333;
    border: 1px solid #c0c0c0;
    border-radius: 3px;
    padding: 6px;
    min-height: 25px;
}
QPushButton:hover {
    background-color: #d0d0d0;
}
QPushButton#applyButton {
    background-color: #4caf50;
    color: white;
    font-weight: bold;
}
QComboBox {
    background-color: #f5f5f5;
    color: #333333;
    border: 1px solid #c0c0c0;
    border-radius: 3px;
    padding: 3px;
    min-height: 25px;
}
QCheckBox {
    color: #333333;
}
QCheckBox::indicator {
    border: 1px solid #c0c0c0;
    border-radius: 2px;
    width: 14px;
    height: 14px;
}
QCheckBox::indicator:checked {
    background-color: #4886e0;
}
QSlider::groove:horizontal {
    background: #d0d0d0;
    height: 6px;
    border-radius: 3px;
}
QSlider::handle:horizontal {
    background: #4886e0;
    width: 14px;
    margin-top: -4px;
    margin-bottom: -4px;
    border-radius: 7px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="detectionGroup">
     <property name="title">
      <string>객체 탐지 설정</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="detectionCheckBox">
        <property name="text">
         <string>객체 탐지 활성화</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="modeLayout">
        <item>
         <widget class="QLabel" name="modeLabel">
          <property name="text">
           <string>탐지 모드:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="modeCombo">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>OpenCV-HOG</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>YOLO</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="yoloGroup">
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>YOLO 설정</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="versionLayout">
        <item>
         <widget class="QLabel" name="versionLabel">
          <property name="text">
           <string>YOLO 버전:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="versionCombo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="modelLayout">
        <item>
         <widget class="QLabel" name="modelLabel">
          <property name="text">
           <string>모델:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="modelCombo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="inputSizeLayout">
        <item>
         <widget class="QLabel" name="inputSizeLabel">
          <property name="text">
           <string>입력 크기:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="inputSizeCombo">
          <item>
           <property name="text">
            <string>416x416</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>608x608</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>832x832</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="confidenceLayout">
        <item>
         <widget class="QLabel" name="confidenceLabel">
          <property name="text">
           <string>신뢰도:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="confidenceSlider">
          <property name="maximum">
           <number>99</number>
          </property>
          <property name="value">
           <number>40</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="confidenceValueLabel">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>0.40</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="currentSettingsGroup">
     <property name="title">
      <string>현재 설정</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="currentSettingsLabel">
        <property name="text">
         <string>탐지기가 설정되지 않았습니다.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>설정 적용</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>닫기</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>